locales/en-US/app.ftl

```diff
@@ -14,16 +14,31 @@
 -firefox-brand-name = Firefox
 -profiler-brand-name = Firefox Profiler
 
-##
+## AppHeader
 
 AppHeader--app-header = <header>{ -profiler-brand-name }</header> — <subheader>Web app for { -firefox-brand-name } performance analysis</subheader>
 AppHeader--github-icon =
     .title = Go to our git repository (this opens in a new window)
 
+## CompareHome
+
+CompareHome--instruction-title = Enter the profile URLs that you’d like to compare
+CompareHome--instruction-content = 
+    The tool will extract the data from the selected track and range for
+    each profile, and put them both on the same view to make them easy to
+    compare.
+
+CompareHome--form-label-profile1 = Profile 1:
+CompareHome--form-label-profile2 = Profile 2:
```

Comment:
@canova I'm not sure if it's the proper way to make these localizable.

Reply:
I think it's fine to keep them that way.